src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int indexBall = new Random().nextInt(100);
```

Comment:
Creating a new instance of 'Random' every time 'getRandomBall' is called can be inefficient. Consider creating a single instance of 'Random' as a class member and reusing it.

Reply:
Fixed